python-sdk/src/astro/databases/__init__.py

```diff
@@ -19,17 +21,21 @@
 SUPPORTED_DATABASES = set(DEFAULT_CONN_TYPE_TO_MODULE_PATH.keys())
 
 
-def create_database(conn_id: str) -> "BaseDatabase":
+def create_database(
+    conn_id: str,
+    table: object | None = None,
+) -> BaseDatabase:
```

Comment:
Can you verify that this does not introduce cyclic dependency or increase import time that we fixed in https://github.com/astronomer/astro-sdk/pull/1013

Reply:
you can use one of the following for it:

- https://docs.python.org/3/library/profile.html
- https://github.com/nschloe/tuna
- https://github.com/ymichael/cprofilev

Reply:
I tried to get the import time for `astro-sdk/python-sdk/src/astro/databases/__init__.py`

<img width="1221" alt="Screenshot 2022-10-26 at 2 26 30 PM" src="https://user-images.githubusercontent.com/98807258/197982565-b9c3784f-3c2e-4eff-929c-241fcfe3155b.png">